modules/core/arrow-data/src/test/kotlin/arrow/data/KleisliTest.kt

```diff
@@ -17,19 +28,34 @@ import io.kotlintest.KTestJUnitRunner
 import io.kotlintest.matchers.shouldBe
 import org.junit.runner.RunWith
 
+@Suppress("TestFunctionName")
```

Comment:
What is this annotation for?

Reply:
Otherwise you get the lint warning about the test names

![captura de pantalla 2018-11-11 a las 18 02 21](https://user-images.githubusercontent.com/6547526/48315867-fbce4d00-e5db-11e8-953f-eaa790480d53.png)